### ISSUE TITLE
Implement Age 9 cards for Artifacts Expansion

### ISSUE THREAD

**Issue by micahstairs** [micahstairs/bga-innovation#146]
This issue will track the implementation of all Age 9 cards for the Artifacts expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

  - [x] **195 - Yeager's Bell X-1A**
  - [x] **196 - Luna 3**
  - [x] **197 - United Nations Charter**
  - [x] **198 - Velcro Shoes**
  - [x] **199 - Philips Compact Cassette**
  - [x] **200 - Syncom 3**
  - [x] **201 - Rock Around the Clock**
  - [x] **202 - Magnavox Odyssey**
  - [x] **203 - The Big Bang**
  - [x] **204 - Marilyn Diptych**
